components/Avatar/Avatar.jsx

```diff
@@ -3,95 +3,81 @@ import { useSupabaseClient } from "@supabase/auth-helpers-react"
 import IconUpload from "@/icons/upload.svg"
 import styles from "./Avatar.module.css"
 import cn from "classnames"
+import { v4 as uuidv4 } from "uuid"
+import AddImage from "../AddImage/AddImage"
 
-export default function Avatar({ uid, url, size, onUpload }) {
-  const supabase = useSupabaseClient()
-  const [avatarUrl, setAvatarUrl] = useState(null)
-  const [uploading, setUploading] = useState(false)
+export default function Avatar({ url, size }) {
+  // const supabase = useSupabaseClient()
+  // const [imageUrl, setImageUrl] = useState(null)
+  // const [uploading, setUploading] = useState(false)
```

Comment:
Do we need to keep things that are commented out? You can always find this code in a previous commit if you need it.